common/traits/wc_race_traits.txt

```diff
@@ -718,6 +718,66 @@ creature_lich = {
 	random = no
```

Comment:
These 2 event texts don't fit the window. Need to either shorten them or expand the window.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/79702908-3a7c3280-82b9-11ea-85c5-40e8f259a2b5.png)
![image](https://user-images.githubusercontent.com/46384992/79702911-3ea85000-82b9-11ea-9da4-834ee7b03950.png)


</details>

Reply:
trimmed, but can't verify at the moment